internal/server/v1beta1/server.go

```diff
@@ -56,6 +56,9 @@ func (server *APIServer) validateUserInCtx(ctx context.Context) (string, error)
 		if errors.Is(err, user.ErrNoUserInformation) {
 			return "", status.Errorf(codes.InvalidArgument, err.Error())
 		}
+		if errors.Is(err, user.DuplicateRecordError{UUID: usr.UUID, Email: usr.Email}) {
```

Comment:
`user.DuplicateRecordError` is not a sentinel error, so better to use `As` rather than `Is` to check.

See https://go.dev/blog/go1.13-errors